Pipfile

```diff
@@ -50,6 +50,7 @@ uritemplate = "==3.0.0"
 urllib3 = "==1.24.1"
 vine = "==1.3.0"
 whitenoise = "==4.1.2"
+readtime = "*"
```

Comment:
Add a version for the readtime package to prevent Pipfile install issues

Reply:
Done.